src/coreclr/vm/zapsig.cpp

```diff
@@ -12,6 +12,7 @@
 // ===========================================================================
 
 
+
```

Comment:
Remove this extra line before checkin. (It's here to convince CI to run the right set of tests.)